### PR TITLE
Peter Grandstaff: Intro of conference co-chair

### PR DESCRIPTION
**Pull request by jcjudkins**
- Added interview post for Peter Grandstaff (Intro as conference co-chair)

### CODE REVIEW

File: _posts/2023-06-26-peter-grandstaff-introduction.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+category: General
+date: 2023-06-26 12:00:00
+image: /static/img/organizers/peter-grandstaff.jpg
+layout: post
+post_photo_alt: Peter Grandstaff, DjangoCon US 2023 Conference Co-Chair
+post_photo_url: /static/img/organizers/peter-grandstaff.jpg
+title: Introducing Peter Grandstaff, DjangoCon US 2023 Conference Co-Chair
+---
+
+This week I had the pleasure of chatting with one of the DjangoCon US 2023 Conference Co-Chairs, Peter Grandstaff, about her role as a conference chair and his experience with Django.
```
Comment (emullaney) [inline _posts/2023-06-26-peter-grandstaff-introduction.md:12]: about "his" role 

(probably a copy paste error from a previous post?)

File: _posts/2023-06-26-peter-grandstaff-introduction.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+## A topic that is very dear to me currently: Any advice for those new to software development?
+
+> Peter: "I think the best thing a new developer can do is to complete an entire project. Ideally, you'd find something you get paid to build, but sometimes it has to be a passion project of your own. The point is to take it all the way from idea to complete product that real people can actually interact with. You're going to learn so much solving all the little problems along the way and you're going to have something real to show for it at the end. Just, don't write it in PHP without a framework like I did. Use a framework. Use Django."
+
+## Thank you so much for your taking the time to chat with me, Peter! What's the best way for people to contact you if they'd like to know more?
+
+> Peter: "The DjangoCon US organizers slack is a great place to find me, or you can find more info and contact information on the Two Rock Software website, [Two Rock](https://tworock.io)"
+
+### We organizers would like to thank Peter for her leadership and work with the planning of DjangoCon US 2023!
```
Comment (emullaney) [inline _posts/2023-06-26-peter-grandstaff-introduction.md:64]: s/her/his

**Review by emullaney — CHANGES_REQUESTED** [review]
Looks really good! Caught two gender errors (probably copy pasta) and commented.

Also, I love this quote from Peter and it would be a fun pull quote for sharing on socials!
>Just, don't write it in PHP without a framework like I did. Use a framework. Use Django

File: _posts/2023-06-26-peter-grandstaff-introduction.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+## Can you tell us about your experience with Django and what inspired you to become involved with the Django community?
+
+> Peter: "I found Django around 2010 after having built and launched a big project where I wrote everything in PHP without using any sort of framework. I knew that I'd built something cool but that I'd taken a very inefficient path to get there. In researching how I should approach that kind of project in the future I came across Django and just really resonated with its philosophy and architecture. Luckily, another project came along soon after and I was able to put it to use pretty quickly.
+
+> I was pretty absent from the Django community despite working with the framework daily for the next decade.
+
+>It was in 2020 when I got to work with [Dawn Wages](http://dawnwages.info/) that I first heard some really wonderful accounts of DjangoCon US and how amazing the Django community was. I attended online in 2021 and 2022 and was pretty impressed.
```
Comment (drewbrew) [inline _posts/2023-06-26-peter-grandstaff-introduction.md:28]:
```suggestion
>It was in 2020 when I got to work with DCUS Sponsorship co-chair [Dawn Wages](http://dawnwages.info/) that I first heard some really wonderful accounts of DjangoCon US and how amazing the Django community was. I attended online in 2021 and 2022 and was pretty impressed.
```

File: _posts/2023-06-26-peter-grandstaff-introduction.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+>As we came out of the pandemic and I learned that [DEFNA](https://www.defna.org/) was looking for a new host city for 2023, I dove in (again, thanks to Dawn) to try to bring the conference here to Durham and I just keep getting deeper into it as we go!"
+
+## What motivated you to apply for the role of DjangoCon US conference chair, and what do you hope to achieve in this position?
+
+> Peter: "Prior to the organizing kickoff for 2023 I'd already taken on some level of leadership by hammering out all the venue and hotel contracts here in Durham. It kind of felt like a natural continuation of what I'd been doing. I'm a DjangoCon US newbie, so I was definitely reluctant to take a leadership role like that. Once I saw how many passionate and engaged volunteers we had showing up to the organizer meetings I felt confident that the Chair role was something I could manage. At least, I could manage it with a good co-chair as backup. Thankfully, the very talented Adrienne Franke, [the other co-chair] agreed to take on that role."
```
Comment (drewbrew) [inline _posts/2023-06-26-peter-grandstaff-introduction.md:34]:
```suggestion
> Peter: "Prior to the organizing kickoff for 2023 I'd already taken on some level of leadership by hammering out all the venue and hotel contracts here in Durham. It kind of felt like a natural continuation of what I'd been doing. I'm a DjangoCon US newbie, so I was definitely reluctant to take a leadership role like that. Once I saw how many passionate and engaged volunteers we had showing up to the organizer meetings I felt confident that the Chair role was something I could manage. At least, I could manage it with a good co-chair as backup. Thankfully, the very talented Adrienne Franke [the other co-chair] agreed to take on that role."
```

File: _posts/2023-06-26-peter-grandstaff-introduction.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+> Peter: "I'm just really blown away by the people involved in organizing this conference. It's a tremendous experience to work with these people."
+
+## A topic that is very dear to me currently: Any advice for those new to software development?
+
+> Peter: "I think the best thing a new developer can do is to complete an entire project. Ideally, you'd find something you get paid to build, but sometimes it has to be a passion project of your own. The point is to take it all the way from idea to complete product that real people can actually interact with. You're going to learn so much solving all the little problems along the way and you're going to have something real to show for it at the end. Just, don't write it in PHP without a framework like I did. Use a framework. Use Django."
+
+## Thank you so much for your taking the time to chat with me, Peter! What's the best way for people to contact you if they'd like to know more?
+
+> Peter: "The DjangoCon US organizers slack is a great place to find me, or you can find more info and contact information on the Two Rock Software website, [Two Rock](https://tworock.io)"
```
Comment (drewbrew) [inline _posts/2023-06-26-peter-grandstaff-introduction.md:62]:
```suggestion
> Peter: "The DjangoCon US organizers Slack is a great place to find me, or you can find more info and contact information on the Two Rock Software website, [Two Rock](https://tworock.io)"
```

File: _posts/2023-06-26-peter-grandstaff-introduction.md [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+## Is there anything that you would like to talk about? 
+
+> Peter: "I'm just really blown away by the people involved in organizing this conference. It's a tremendous experience to work with these people."
+
+## A topic that is very dear to me currently: Any advice for those new to software development?
+
+> Peter: "I think the best thing a new developer can do is to complete an entire project. Ideally, you'd find something you get paid to build, but sometimes it has to be a passion project of your own. The point is to take it all the way from idea to complete product that real people can actually interact with. You're going to learn so much solving all the little problems along the way and you're going to have something real to show for it at the end. Just, don't write it in PHP without a framework like I did. Use a framework. Use Django."
```
Comment (drewbrew) [inline _posts/2023-06-26-peter-grandstaff-introduction.md:58]: As someone who ported a homegrown lamp stack to Django, I feel your pain

**Review by drewbrew — APPROVED** [review]

**Comment by PeterGrand** [comment]
I don't really know what's normal here, but I've been referring to myself as Chair and Adrienne as Co-Chair, which in my head has been sort like President and Vice President. Happy to go with whatever nomenclature is normal for these, though.

File: _posts/2023-06-26-peter-grandstaff-introduction.md (hunk 2 above)
Comment (jcjudkins) [reply _posts/2023-06-26-peter-grandstaff-introduction.md:64]: 🤦

File: _posts/2023-06-26-peter-grandstaff-introduction.md (hunk 1 above)
Comment (jcjudkins) [reply _posts/2023-06-26-peter-grandstaff-introduction.md:12]: Indeed!

**Comment by jcjudkins** [comment]
> I don't really know what's normal here, but I've been referring to myself as Chair and Adrienne as Co-Chair, which in my head has been sort like President and Vice President. Happy to go with whatever nomenclature is normal for these, though.

I did in fact know this! And forgot to update!

**Review by drewbrew — APPROVED** [review]

**Pull request merged**